bench/src/test/scala/benchmark/rabenchmarks.scala

```diff
@@ -1,9 +1,8 @@
 package cell
 
-import lattice.{ Lattice, NaturalNumberLattice, NaturalNumberKey }
+import lattice._
```

Comment:
Please revert this change. Wildcard imports are a suboptimal default, and there is no good reason to prefer a wildcard import in this case.